src/utils/KeyBindingUtils.ts

```diff
@@ -252,6 +253,16 @@ export const DefaultKeyBindings: Record<KeyBindingType, [KeyCombination | null,
       key: "B",
     },
   ],
+  [ScriptEditorAction.Run]: [
+    {
+      control: true,
+      alt: false,
+      shift: false,
+      meta: false,
+      key: "Q",
```

Comment:
I looked to see if there is any kind of standard here. Curiously, most IDEs (IDEA, Eclipse, VS) have settled on some form of function key for run, but they don't agree on which one. (F5, F10 and F11 are all used.) So, I think this is probably fine.